instrumentation/servlet/servlet-common/src/main/java/io/opentelemetry/instrumentation/auto/servlet/dispatcher/RequestDispatcherInstrumentation.java

```diff
@@ -115,18 +114,16 @@ public static void start(
         parent = servletContext;
       }
 
-      try (Scope ignored = ContextUtils.withScopedContext(parent)) {
-        span = TRACER.startSpan(method);
+      span = TRACER.startSpan(method);
```

Comment:
Here the correct parent span is important. I think this change breaks it.

Reply:
Eeek, thanks for catching this!

Reply:
Can we add a test for it?

Reply:
done